### PR TITLE
fix fp8_paged_mqa_logits output, support default cuda graph batch size

### PR DESCRIPTION
**Pull request by wufann**
## Motivation
Remove --cuda-graph-max-bs 64 limit

## Modifications
logits.sum(dim=0) this operation has already done within the kernel.

## Accuracy Tests
```
server:
python3 -m sglang.launch_server --model /models/DeepSeek-V3.2-Exp \
        --tp 8 \
        --port 10086 \
        --trust-remote-code \
        --nsa-prefill-backend tilelang \
        --nsa-decode-backend tilelang

client:
python3 bench_sglang.py --num-questions 1319 --port 10086 --parallel 96

Accuracy: 0.961
Invalid: 0.000
Latency: 262.738 s
Output throughput: 461.950 token/s
```

## Checklist

- [ ] Format your code according to the [Format code with pre-commit](https://docs.sglang.io/developer_guide/contribution_guide.html#format-code-with-pre-commit).
- [ ] Add unit tests according to the [Run and add unit tests](https://docs.sglang.io/developer_guide/contribution_guide.html#run-and-add-unit-tests).
- [ ] Update documentation according to [Write documentations](https://docs.sglang.io/developer_guide/contribution_guide.html#write-documentations).
- [ ] Provide accuracy and speed benchmark results according to [Test the accuracy](https://docs.sglang.io/developer_guide/contribution_guide.html#test-the-accuracy) and [Benchmark the speed](https://docs.sglang.io/developer_guide/contribution_guide.html#benchmark-the-speed).
- [ ] Follow the SGLang code style [guidance](https://docs.sglang.io/developer_guide/contribution_guide.html#code-style-guidance).

## Review Process

1. Ping Merge Oncalls to start the PR flow. See the [PR Merge Process](https://github.com/sgl-project/sglang/blob/main/.github/MAINTAINER.md#pull-request-merge-process).
2. Get approvals from [CODEOWNERS](https://github.com/sgl-project/sglang/blob/main/.github/CODEOWNERS) and other reviewers.
3. Trigger CI tests with [comments](https://docs.sglang.io/developer_guide/contribution_guide.html#how-to-trigger-ci-tests) or contact authorized users to do so.
   - `/tag-run-ci-label`, `/rerun-failed-ci`, `/tag-and-rerun-ci`
4. After green CI and required approvals, ask Merge Oncalls to merge.

### CODE REVIEW

**Review by hubertlu-tw — APPROVED** [review]
LGTM. Thanks!
